### DISCUSSION TITLE
Request for Customizable Date Format (DD/MM) in Daily Column Header

### DISCUSSION THREAD

**Discussion by 0rteip** [davish/obsidian-full-calendar#515] [Feature Requests]
### ⚠️ Please check that this feature request hasn't been suggested before.

- [X] I searched previous [Ideas in Discussions](https://github.com/davish/obsidian-full-calendar/discussions/categories/feature-requests) didn't find any similar feature requests.
- [X] I searched previous [Issues](https://github.com/davish/obsidian-full-calendar/issues) didn't find any similar feature requests.

### 🔖 Feature description

I would like to request the ability to customize the date format in the daily column header. Currently, the format is MM/DD, but it would be beneficial if users could have the option to change it to DD/MM. Maybe even the calendar page header could be customizable.

### ✔️ Solution

Implementing a setting that allows users to choose their preferred date format like in the Daily notes Core plugins.

**Comment by arthurmougin** [comment]
I am also interested in this feature as an european

**Reply by arthurmougin** [reply]
This format would be nice across the whole calendar
